Source/Project64/Multilanguage/Language Class.cpp

```diff
@@ -269,13 +270,16 @@ void CLanguage::LoadDefaultStrings (void)
 	DEF_STR(ROM_USE_TLB,         L"Use TLB");
 	DEF_STR(ROM_REG_CACHE,       L"Register caching");
 	DEF_STR(ROM_DELAY_SI,        L"Delay SI Interrupt");
-	DEF_STR(ROM_SP_HACK,         L"SP Hack");
+	DEF_STR(ROM_VIREFRESH,       L"Vi Refresh Rate: ");

```

Comment:
All I know is in PJ64 1.6, it was called "SP Hack", and in 1.7.0.50+, it's renamed to "Fast SP".

So provided what MELERIX says is true, then the macro names were backwards from the text string names in the GUI when running the application.